lens-labels/src/Lens/Labels.hs

```diff
@@ -81,8 +84,12 @@ class HasLens (x :: Symbol) f s t a b
 instance
     (p ~ (a -> f b), q ~ (s -> f t), HasLens x f s t a b)
     => IsLabel x (LensFn p q) where
+#if __GLASGOW_HASKELL__ >= 802
+    fromLabel = LensFn $ lensOf (proxy# :: Proxy# x)
```

Comment:
It's `-XMagicHash`:
https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/glasgow_exts.html#the-magic-hash